coq/coq-system.el

```diff
@@ -378,12 +378,26 @@ options of a few coq-project files does the right thing."
 LOADPATH, CURRENT-DIRECTORY, PRE-V85: see `coq-include-options'."
   (coq-include-options loadpath current-directory pre-v85))
 
+(defun coq-clean-prog-args (args)
+  "Return a copy of ARGS with automatically added entries of `coq-prog-args' removed.
```

Comment:
Thanks for having documented this new function. Two related suggestions:

* Change the first line to
    ```suggestion
  "Return ARGS without the entries added by `coq-coqtop-prog-args'.
    ```
    to cope with usual [guidelines for elisp docstrings](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html):
    > Format the documentation string so that it fits in an Emacs window on an 80-column screen. It is a good idea for most lines to be no wider than 60 characters. The first line should not be wider than 67 characters or it will look bad in the output of apropos. 
    > (…)

* Add a double-hyphen in the identifier of this new function (even this convention is not yet widely used in PG), e.g. `coq--clean-prog-args`.  
    Cf. [this page](https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html):
    > Use two hyphens to separate prefix and name if the symbol is not meant to be used by other packages. (…)

Reply:
Changed